com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/ShaderVariables.hlsl

```diff
@@ -54,7 +54,47 @@
 #define RAY_TRACING_OPTIONAL_ALPHA_TEST_PASS
 #endif
 
+// ----------------------------------------------------------------------------
```

Comment:
guess we should move to common.hlsl instead. Don't think shadervariables make sense. Thought?

Reply:
The problem with Common is that it is shared across SRPs and the way we scalarize now is very HDRP specific; I am worried that a generic SRP might misinterpret those functions (which are not overly generic)

Reply:
ok fair enough. Then can you move it to shadervariablesFunction.hlsl? :) thanks

Reply:
Done